components/Person.tsx

```diff
@@ -0,0 +1,18 @@
+import { User } from '@geist-ui/react';
+
+import { Owner } from '../redux/types';
+
+interface PersonParams {
+  person: Owner;
+  onClick?: (name: string, filter: Owner) => void;
+}
+
+const Person = ({ person, onClick }: PersonParams): JSX.Element => (
+  <User
+    name={person.name}
+    onClick={(): void => onClick && onClick('owners', person)}
```

Comment:
Yeah! optional chaining is that good 💯
```suggestion
    onClick={(): void => onClick?.('owners', person)}
```

Reply:
I didn't think you could do optional chaining on method calls, can you?